Makefile

```diff
@@ -97,8 +100,15 @@ build_dir := $(DIR)/.build/$(os1)-$(arch1)
 go_version_full := $(shell cat .go-version)
 go_version := $(go_version_full:.0=)
 go_dir := $(build_dir)/go/$(go_version)
-go_bin_dir := $(go_dir)/bin
-go_url = https://storage.googleapis.com/golang/go$(go_version).$(os1)-$(arch2).tar.gz
+
+ifeq ($(os1),windows)
+	go_bin_dir = $(go_dir)/go/bin
```

Comment:
Why does this have to be different?

Reply:
windows golang zip does not contains files on base, but they put all files inside `go` folder you can take a look to it downloading for example https://go.dev/dl/go1.17.4.windows-amd64.zip

Reply:
👍 ok, sounds good